OpenSim/Simulation/Model/AbstractTool.cpp

```diff
@@ -110,6 +110,10 @@ AbstractTool::AbstractTool(const string &aFileName, bool aUpdateFromXMLNode):
     _analysisSet.setMemoryOwner(false);
     setNull();
     if(aUpdateFromXMLNode) updateFromXMLDocument();
+
+    if(_modelFile.empty())
```

Comment:
we can now use `OPENSIM_THROW_IF_FRMOBJ` and the message could be a bit clearer, e.g:
*"No model file was specified (<model_file> tag is empty) in the Tool's Setup file."*